solr/test-framework/src/java/org/apache/solr/util/BadHdfsThreadsFilter.java

```diff
@@ -41,6 +41,10 @@ public boolean reject(Thread t) {
       return true;
     } else if (name.startsWith("nioEventLoopGroup")) {
       return true; // netty threads waiting for a web server stop confirm that won't happen
+    } else if (name.startsWith("globalEventExecutor")) { // HADOOP-11245
```

Comment:
This JIRA claims it is only in 3.4.0, not 3.3.x?

Reply:
yea so I agree with you - I don't quite understand. I need to dig a bit more into this. It definitely seems related to Netty 4 in Hadoop though.

Reply:
Updated the comment to point to Netty 4 Hadoop jira.